redshift_sqlalchemy/models.py

```diff
@@ -0,0 +1,55 @@
+"""
+System Tables and Views
+
+Amazon Redshift has many system tables and views that contain information
+about how the system is functioning. This module contains SQLAlchemy
+Declarative models for these tables.
+"""
+
+
+import sqlalchemy as sa
+from sqlalchemy.ext import declarative
+
+
+Base = declarative.declarative_base()
+
+
+class STLUnloadLog(Base):
+    """
+    Records the details for an unload operation.
+
+    STL_UNLOAD_LOG records one row for each file created by an UNLOAD
+    statement. For example, if an UNLOAD creates 12 files, STL_UNLOAD_LOG will
+    contain 12 corresponding rows.
+
+    This table is visible to all users.
+    """
+
+    __tablename__ = 'STL_UNLOAD_LOG'
+
+    user_id = sa.Column('userid', sa.Integer())
+    """ID of the user who generated the entry."""
+
+    query_id = sa.Column('query', sa.Integer(), primary_key=True)
+    """ID for the transaction."""
```

Comment:
I'd almost be inclined to define all columns as part of the primary key. That would at least be a consistent pattern that avoids us having to make an arbitrary choice for each new table we model.